### PR TITLE
iex: Autocomplete Erlang callbacks

### PR DESCRIPTION
**Pull request by wojtekmach**

### CODE REVIEW

File: lib/iex/lib/iex/autocomplete.ex [hunk 1]
```diff
@@ -459,15 +459,20 @@ defmodule IEx.Autocomplete do
   end
 
   defp get_module_callbacks(mod) do
-    cond do
```
Comment (fertapric) [inline lib/iex/lib/iex/autocomplete.ex]: Nitpick: you can use `with` here

```elixir
defp get_module_callbacks(mod) do
  with true <- ensure_loaded?(mod),
       {:ok, callbacks} <- Code.Typespec.fetch_callbacks(mod) do
    for {name_arity, _} <- callbacks do
      {_kind, name, arity} = IEx.Introspection.translate_callback_name_arity(name_arity)
      {name, arity}
    end
  else
    _ -> []
  end
end
```

Comment (wojtekmach) [reply lib/iex/lib/iex/autocomplete.ex]: @fertapric see https://github.com/elixir-lang/elixir/pull/9110#discussion-diff-290219277R448 :D

Comment (fertapric) [reply lib/iex/lib/iex/autocomplete.ex]: 😅 Ok, thanks!

**Comment by fertapric** [comment]
Thanks @wojtekmach! 💜

**Pull request merged**